src/Generating/DungeonRoomsFinisher.cpp

```diff
@@ -292,7 +292,7 @@ cDungeonRoomsFinisher::cDungeonRoomsFinisher(cTerrainShapeGen & a_ShapeGen, int
 	m_ShapeGen(a_ShapeGen),
 	m_MaxHalfSize((a_MaxSize + 1) / 2),
 	m_MinHalfSize((a_MinSize + 1) / 2),
-	m_HeightProbability(cChunkDef::Height)
+	m_HeightProbability(cChunkDef::UpperLimit)
```

Comment:
:exclamation:  Anything using this needs to be reworked, `m_HeightProbability` doesn't support negative values.